ce_default_content/content/file/a971a73c-0473-4d72-909e-b4ced94969e4.yml

```diff
@@ -0,0 +1,27 @@
+_meta:
+  version: '1.0'
+  entity_type: file
+  uuid: a971a73c-0473-4d72-909e-b4ced94969e4
+  default_langcode: en
+default:
+  uid:
+    -
+      target_id: 1
+  filename:
+    -
+      value: gippity-sauce-factory.jpeg
+  uri:
+    -
+      value: 'public://2024-02/gippity-sauce-factory.jpeg'
```

Comment:
Can you move all the images to ce_media, as they are default or testing images.

Reply:
I think we should keep it separate from ce_media as current because ce_media should only build structure.